grails-app/controllers/org/pih/warehouse/api/SelectOptionsApiController.groovy

```diff
@@ -79,12 +80,24 @@ class SelectOptionsApiController {
     }
 
     def preferenceTypeOptions() {
+        List<Map<String, String>> defaultPreferenceTypes = [
+            [
+                id: ProductSupplierService.PREFERENCE_TYPE_MULTIPLE,
+                label: g.message(code: "react.productSupplier.preferenceType.multiple.label", default: "Multiple")
+            ],
+            [
+                id: ProductSupplierService.PREFERENCE_TYPE_NONE,
+                label: g.message(code: 'react.productSupplier.preferenceType.none.label', default: "None")
+            ],
+        ]
```

Comment:
I decided to add the default options here, not on the frontend side, to avoid potential problems with translations that we've had for statuses options once (it was due to the fact that the labels had been rendered before the translation plugin or the redux's current locale was injected).
cc @jmiranda